crowbar_framework/spec/controllers/api/upgrade_controller_spec.rb

```diff
@@ -57,6 +57,24 @@
       expect(response).to have_http_status(:unprocessable_entity)
     end
 
+    it "initiates the upgrade of nodes" do
+      allow(NodeObject).to(
+        receive(:find).and_return([NodeObject.find_node_by_name("testing")])
+      )
+      post "/api/upgrade/nodes", {}, headers
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to eq("true")
```

Comment:
as mentioned in the comment above, i think we dont need to check for the body here